config/sitemap.rb

```diff
@@ -0,0 +1,14 @@
+Sitemap::Generator.instance.load(host: 'era.library.ualberta.ca') do
+  path :root, priority: 1, change_frequency: 'weekly'
+  path :catalog_index, priority: 1, change_frequency: 'weekly'
+  User.all.each do |user|
+    literal Sufia::Engine.routes.url_helpers.profile_path(user.to_param), priority: 0.8, change_frequency: 'daily'
+  end
+  read_group = Solrizer.solr_name('read_access_group', :symbol)
+  GenericFile.where(read_group => 'public').each do |f|
```

Comment:
Pulling in the full AF object just to get the url is very expensive - could we make this a solr-only operation? I'm afraid in production this will be like the audit process and take days to run. @pgwillia